lib/pip.js

```diff
@@ -87,19 +86,17 @@ function installRequirements(requirementsPath, targetFolder, serverless, service
       cmdOptions.push('-e', 'SSH_AUTH_SOCK=/tmp/ssh_sock');
     }
     if (process.platform === 'linux') {
-      // Set the ownership of the .serverless/requirements folder to current user
-      pipCmd = quote(pipCmd);
-      const chownCmd = quote([
-        'chown', '-R', `${process.getuid()}:${process.getgid()}`,
-        targetRequirementsFolder,
-      ]);
-      pipCmd = ['/bin/bash', '-c', '"' + pipCmd + ' && ' + chownCmd + '"'];
+      // Use same user so requirements folder is not root and so --cache-dir works
+      cmdOptions.push('-u', `${process.getuid()}`);
```

Comment:
@cgrimal, AFAICT you added the `chown` stuff. Could you test this alternative?

Reply:
FWIW tests passed on Linux with all Docker tests enabled. There was no problem deleting the `.serverless` folder.